mipengine/common/common_data_elements.py

```diff
@@ -53,18 +56,23 @@ def __iter__(self):
 
 @dataclass
 class CommonDataElement:
```

Comment:
CommonDataElement and MetadataVariable have almost the same fields. What is their difference?

Reply:
The metadata classes, MetadataVariable and MetadataGroup are used to parse the metadata.json coming from data catalogue.

The CommonDataElement classes are used to convert it in a more readable format that we are going to use internally in the MIP engine.